pool/config.go

```diff
@@ -20,6 +20,8 @@ type Config struct {
 	// its destruction. set 1 to create new process for each new task, 0 to let
 	// worker handle as many tasks as it can.
 	MaxJobs uint64 `mapstructure:"max_jobs"`
+	// TODO Comment
+	MaxJobsJitter bool `mapstructure:"max_jobs_jitter"`
```

Comment:
No need to add an additional option. Use constant Jitter with 0-10% of the initial value randomly. Adding an option is easy, but removing whether it is confusing or not properly described is hard.

Reply:
Although it may be good to make the default behavior. It’s unlikely that someone wants to simultaneously restart processes

Reply:
I thought about this again. How will we test this functionality? Surely the tests check the execution of tasks without jitter. A random jitter will skew the results, so I'm inclined to be able to turn this functionality on or off.

Reply:
Check for the number of Execs which should be > than `max_jobs`, but less, than 10% (maximum random value which might be assigned).
For example: max_jobs: 100, max execs with jitter should be in the range of (100-110]